### PR TITLE
Allow to configure multiple rpc endpoints

### PR DESCRIPTION
**Pull request by smallhive**

### CODE REVIEW

File: api/resolver/init.go [hunk 1]
```diff
@@ -17,7 +17,12 @@ import (
 // The Container should be used like regular [NNSResolver].
 type Container struct {
 	mu       sync.RWMutex
-	resolver *NNSResolver
+	resolver NNSResolver
+}
+
+// NNSResolver allows to resolve container id by its name.
+type NNSResolver interface {
```
Comment (roman-khimov) [inline api/resolver/init.go:24]: Interface with the only implementation in the same package can't exist.

File: config/config.yaml [hunk 2]
```diff
@@ -49,7 +49,9 @@ tree:
   service: node1.neofs:8080
 
 # RPC endpoint and order of resolving of bucket names
-rpc_endpoint: http://morph-chain.neofs.devenv:30333
+morph:
```
Comment (roman-khimov) [inline config/config.yaml:52]: Let's call this `fschain` immediately. We will eventually remove `morph` from configs like this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,9 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+### Changed
+- Single `rpc_endpoint` config option replaced with multiple endpoints `morph.endpoints` option (#968)
+
```
Comment (roman-khimov) [inline CHANGELOG.md:9]: Migration guide is needed.

Comment (smallhive) [reply CHANGELOG.md:23]: Added

File: api/resolver/init.go (hunk 1 above)
Comment (smallhive) [reply api/resolver/init.go]: Sure, it was too much

File: config/config.yaml (hunk 2 above)
Comment (smallhive) [reply config/config.yaml]: Renamed

**Review by roman-khimov — APPROVED** [review]
I'm not excited about error handling here (if case some node is down), but it's already something.

**Pull request merged**